docs/guides/Localization and Internationalization.md

```diff
@@ -27,39 +27,43 @@ func mapView(_ mapView: MGLMapView, didFinishLoading style: MGLStyle) {
 
 By default, distances are given in the predominant measurement system of the system region, which may not necessarily be the same region in which the user is traveling. To override the measurement system used in spoken instructions, set the `RouteOptions.distanceMeasurementSystem` property when calculating the route with which to start navigation.
 
 The upcoming road or ramp destination is named according to the local or national language. In some regions, the name may be given in multiple languages.
 
 ## Supported languages
 
-| Language   | User interface | [Spoken instructions][osrmti] | Mapbox Voice API | [`AVSpeechSynthesizer`][iossynth]<br>(iOS 11)
-|------------|:--------------:|:-----------------------------:|:---------------------:|:--------------------------------:
-| Arabic    | ✅              | —                             | —                     | ✅
-| Catalan    | ✅              | —                             | —                     | —
-| Chinese    | ✅ Simplified   | ✅                             | —                     | ✅
-| Danish     | ✅              | ✅                             | ✅                     | ✅
-| Dutch      | ✅              | ✅                             | ✅                     | ✅
-| English    | ✅              | ✅                             | ✅                     | ✅
-| French     | ✅              | ✅                             | ✅                     | ✅
-| German     | ✅              | ✅                             | ✅                     | ✅
-| Hebrew     | ✅              | ✅                             | —                     | ✅
-| Hungarian  | ✅              | —                             | —                     | ✅
-| Italian    | ✅              | ✅                             | ✅                     | ✅
-| Portuguese | ✅             | ✅                             | ✅                     | ✅
-| Polish     | —              | ✅                             | ✅                     | ✅
-| Romanian   | —              | ✅                             | ✅                     | ✅
-| Russian    | ✅              | ✅                             | ✅                     | ✅
-| Spanish    | ✅              | ✅                             | ✅                     | ✅
-| Swedish    | ✅              | ✅                             | ✅                     | ✅
-| Turkish    | —              | ✅                             | ✅                     | ✅
-| Vietnamese | ✅              | ✅                             | —                     | —
+The table below lists the languages that are supported for user interface elements and for spoken instructions. If a language is marked as “manual”, the user will not automatically receive spoken instructions in that language, so you would need to set `RouteOptions.locale` in order to use it.
```

Comment:
Maybe it would be helpful to outline the process to make:

> If a language is marked as “manual”, the user will not automatically receive spoken instructions in that language, so you would need to set `RouteOptions.locale` in order to use it.

automatic and why this is the case.

Reply:
It isn’t possible to use these particular languages automatically. The top portion of this document does discuss how the other languages are automatic and why these particular languages are manual.